auth-web/src/views/auth/AccountSettings.vue

```diff
@@ -68,6 +81,7 @@ export default class AccountSettings extends Vue {
   private readonly currentUser!: KCUserProfile
   private isLoading = true
   private isDirSearchUser: boolean = false
+  private dirSearchUrl = ConfigHelper.getValue('DIRECTOR_SEARCH_URL')
```

Comment:
We should create a method in ConfigHelper `getDirectorSearchUrl` to be consistent with how we retrieve other URLs.

Reply:
ok..will do it now..

Reply:
Done